core/src/main/scala/kafka/tools/TestRaftServer.scala

```diff
@@ -428,7 +428,7 @@ object TestRaftServer extends Logging {
   }
 
   private class TestRaftServerOptions(args: Array[String]) extends CommandDefaultOptions(args) {
-    val configOpt: OptionSpec[String] = parser.accepts("config", "Required configured file")
+    val configOpt: OptionSpec[String] = parser.accepts("config", "REQUIRED: The configured file")
```

Comment:
Could you please use `CommandLineUtils.checkRequiredArgs` to check the existence in this PR?